docs/development/windows.md

```diff
@@ -134,12 +134,39 @@ npm install
 
 ### Optional: Create `.env` file
 
-Copy the example and fill in secrets (if needed):
+App can be booted without any custom credentials. But if you would like to use services that require custom credentials (e.g. S3, Stripe, Resend, etc.), you can copy the `.env.example` file to `.env` and fill in the values.:
```

Comment:
Please feel free to send a PR to configure S3 bucket name via ENV variable name with a fallback to the hard-coded values.